src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/PdlUtil.kt

```diff
@@ -43,4 +45,18 @@ inline fun <reified T : Any> feilsjekkOgReturnerData(pdlResponse: PdlBolkRespons
         throw PdlRequestException("Feil ved henting av ${T::class} fra PDL. Se secure logg for detaljer.")
     }
     return pdlResponse.data.personBolk.associateBy({ it.ident }, { it.person!! })
+}
+
+fun feilmeldOgReturnerData(pdlResponse: PdlIdentBolkResponse): Map<String, PdlIdent> {
+    if (pdlResponse.data == null) {
+        secureLogger.error("Data fra pdl er null ved bolkoppslag av identer fra PDL: ${pdlResponse.errorMessages()}")
+        throw PdlRequestException("Data er null fra PDL -  ${PdlIdentBolkResponse::class}. Se secure logg for detaljer.")
+    }
+
+    val feil = pdlResponse.data.hentIdenterBolk.filter { it.code != "ok" }.associate { it.ident to it.code }
+    if (feil.isNotEmpty()) {
+        // Logg feil og gå vider. Ved feil returneres nåværende ident.
```

Comment:
Kan evt. skrive denne kommentaren "Ved feil returneres nåværende ident" i logg-meldingen nedenfor? Lett for folk å se/skjønne om 5 år :)